contracts/bonding/BondingManager.sol

```diff
@@ -347,22 +347,22 @@ contract BondingManager is ManagerProxyTarget, IBondingManager {
         // Delegator must either have unbonded tokens or be in the unbonded state
         require(delegators[msg.sender].unbondedAmount > 0 || delegatorStatus(msg.sender) == DelegatorStatus.Unbonded);
 
-        uint256 amount = 0;
-
         if (delegators[msg.sender].unbondedAmount > 0) {
-            // Withdraw unbonded amount
-            amount = amount.add(delegators[msg.sender].unbondedAmount);
+            // Withdraw unbonded amount (ETH)
+            uint256 unbondedAmount = delegators[msg.sender].unbondedAmount;
             delegators[msg.sender].unbondedAmount = 0;
+
+            minter().transferETH(msg.sender, unbondedAmount);
```

Comment:
I'm a little confused about what "withdrawing" after the unbonding period, has to do with ETH. Maybe the term is overloaded...perhaps withdrawing a deposit should be separate than withdrawing after the unbonding period has ended, since one will be withdrawing ETH and the other withdrawing LPT?

Reply:
maybe `withdrawDeposit` and `withdrawStake`?